security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc

```diff
@@ -6,19 +6,22 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The {cert-manager-operator} supports using ACME CA servers, such as Let's Encrypt, to issue certificates.
+The {cert-manager-operator} supports using ACME CA servers, such as Let's Encrypt, to issue certificates. Explicit credentials are configured by specifying the secret details in the `Issuer` API object. Ambient credentials are extracted from the environment, metadata services, or local files which are not explicitly configured in the `Issuer` API object.
 
-// About ACME issuers
-include::modules/cert-manager-acme-about.adoc[leveloffset=+1]
+// Configuring an ACME issuer to solve DNS01 challenges by using explicit credentials on AWS
+include::modules/cert-manager-acme-dns01-explicit-aws.adoc[leveloffset=+1]
 
-// Supported ACME challenges types
-include::modules/cert-manager-acme-challenges-types.adoc[leveloffset=+2]
+//Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on AWS
+include::modules/cert-manager-acme-dns01-ambient-aws.adoc[leveloffset=+1]
 
-// Supported DNS-01 providers
-include::modules/cert-manager-acme-dns-providers.adoc[leveloffset=+2]
+// Configuring an ACME issuer to solve DNS01 challenges by using explicit credentials on GCP
+include::modules/cert-manager-acme-dns01-explicit-gcp.adoc[leveloffset=+1]
 
-// Configuring an ACME issuer to solve HTTP01 challenges
-include::modules/cert-manager-acme-http01.adoc[leveloffset=+1]
+// Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on GCP
+include::modules/cert-manager-acme-dns01-ambient-gcp.adoc[leveloffset=+1]
+
+// Configuring an ACME issuer to solve DNS01 challenges by using explicit credentials on Microsoft Azure
+include::modules/cert-manager-acme-dns01-explicit-azure.adoc[leveloffset=+1]
 
-// Configuring an ACME issuer to solve DNS01 challenges
-include::modules/cert-manager-acme-dns01-aws.adoc[leveloffset=+1]
+// Configuring an ACME issuer to solve HTTP01 challenges
```

Comment:
PTAL if you want to add HTTP-01 first and the remaining DNS-01 challenges later to that (like it was there previously). But I'd leave for you to decide, consider this one optional :)

Reply:
Keeping it like it was there previously